atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/Transaction.java

```diff
@@ -143,6 +149,8 @@
      * ordering of rows is undefined. In case of duplicate rows, the ordering is based on the first occurrence of
      * the row.
      *
+     * Iterators must be used strictly within the scope of the transaction.
```

Comment:
Also, you haven't added a similar comment on `getAsync`!

Reply:
good spot - added!